omnibus/config/software/chef.rb

```diff
@@ -44,6 +44,10 @@
 dependency "chef-gem-byebug"
 dependency "chef-gem-debug_inspector"
 dependency "chef-gem-binding_of_caller"
+unless ios_xr? || solaris?
```

Comment:
We don't need this feature on either Cisco or Solaris, its mostly for workstations, and Cisco's are definitely not workstations, and Solaris workstations should be reasonably miniscule as a use case.  The bigger problem here is Linux, Windows, Mac, and maybe FreeBSD.  It introduces a slight discrepancy in our omnibus philosophy, but we've had that before with ruby-shadow.